.github/workflows/typos.toml

```diff
@@ -1,2 +1,6 @@
 [default.extend-words]
 groth = "groth"
+
+[type.po]
+extend-glob = ["*.circom"]
+check-file = false
```

Comment:
small nit: do we need this? I understand that this was added to prevent lints over the imported vocdoni/keccak256-circom repo files, but I assume that no longer is needed, and it's good to be able to catch typos in the circom files.